### PR TITLE
feat: add chatwoot, replace discord as main help link

### PR DESCRIPTION
**Pull request by kilrau**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -112,5 +112,21 @@
         <script
             src="/node_modules/iframe-resizer/js/iframeResizer.contentWindow.js"
             type="module"></script>
+        <script>
+            (function(d,t) {
+                var BASE_URL="https://support.boltz.exchange";
```
Comment (michael1011) [inline index.html:117]: Should be configurable

**Review by michael1011 — CHANGES_REQUESTED** [review]
The Chatwoot script should be loaded programmatically in the entrypoint of the Solid application *if* its token is set in the `.env` file

**Review by michael1011 — CHANGES_REQUESTED** [review]
GitHub posted the review twice

**Comment by kilrau** [comment]
Can't approve as it's my own PR, but LGTM

File: src/chatwoot.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            };
+        }
+    });
+
+    const { swap } = usePayContext();
+
+    createEffect(() => {
+        if (swap() !== null) {
+            window.$chatwoot.setCustomAttributes({
+                swapId: swap().id,
```
Comment (michael1011) [inline src/chatwoot.ts:49]: Is not propagated to the Chatwoot UI

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: src/chatwoot.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            setCustomAttributes: (attributes: { swapId: string }) => void;
+        };
+    }
+}
+
+export default function Chatwoot() {
+    onMount(() => {
+        const token = import.meta.env.VITE_CHATWOOT_TOKEN;
+        const url = config.chatwootUrl;
+        if (token !== undefined && url !== undefined) {
```
Comment (michael1011) [inline src/chatwoot.ts:24]: Let's also do a `log.warn` in case it's not set

**Pull request merged**